markdown-toc.el

```diff
@@ -101,12 +101,18 @@
        (-repeat it sym)
        (s-join "" it)))
 
+(defconst markdown-toc--protection-symbol "09876543214b825dc642cb6eb9a060e54bf8d69288fbee49041234567890"
```

Comment:
> It's just a long random string, nothing magic? I am curious about this.
@ardumont <https://github.com/ardumont>

yep, something like that.
It's the `git hash-object -t tree /dev/null` (empty directory's hash)
prepended with 0987654321 and appended with 1234567890 ^^

```sh
$ git hash-object -t tree /dev/null
4b825dc642cb6eb9a060e54bf8d69288fbee4904
```

I just wanted to protect the - from being removed ^^
IIRC my mood at that point, i think i did not want to hassle myself with
regexp...
But of course, if something better comes along, i'll take it (it is covered
by tests now ^^).

Cheers,

tony / @ardumont

On Wed, Jul 20, 2016 at 12:53 PM, Merlin Dou <notifications@github.com>
wrote:

> In markdown-toc.el
> <https://github.com/ardumont/markdown-toc/pull/25#discussion_r71503866>:
>
> > @@ -101,12 +101,18 @@
> >         (-repeat it sym)
> >         (s-join "" it)))
> >
> > +(defconst markdown-toc--protection-symbol "09876543214b825dc642cb6eb9a060e54bf8d69288fbee49041234567890"
>
> It's just a long random string, nothing magic? I am curious about this.
> @ardumont <https://github.com/ardumont>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ardumont/markdown-toc/pull/25/files/50bf208cfd20242a7b122585f922baa83fbd3598#r71503866>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAr33GKbARyLUpx6s3855bdyLaBfRnSgks5qXf4UgaJpZM4JQmdc>
> .
>